rbac/management/group/view.py

```diff
@@ -360,7 +385,8 @@ def principals(self, request, uuid=None):
             if isinstance(resp, dict) and 'errors' in resp:
                 return Response(status=resp['status_code'], data=resp['errors'])
             output = GroupSerializer(resp)
-            return Response(status=status.HTTP_200_OK, data=output.data)
+        elif request.method == 'GET':
+            output = PrincpalSerializer(group.principals.all(), many=True)
```

Comment:
I'm guessing we'll probably want to have this data be paginated like all other (apart from `/access/`) endpoints?

Reply:
Yeah, I went back and forth on that. Wasn't sure if it was worth paginating a bare list of Principals, but I can do that, no problem!